README.md

```diff
@@ -80,18 +77,21 @@ The HTTP library can be used on the browser via the [BrowserClient][] class in
 XMLHttpRequests. For example:
 
 ```dart
+import 'dart:async';
 import 'package:http/browser_client.dart';
-import 'package:http/http.dart' as http;
 
-var client = new BrowserClient();
-var url = "/whatsit/create";
-client.post(url, body: {"name": "doodle", "color": "blue"})
-    .then((response) {
-  print("Response status: ${response.statusCode}");
-  print("Response body: ${response.body}");
-});
+Future<Null> main() async {
```

Comment:
We haven't yet adopted that convention for our packages, yet.

For now, remove the annotation. Should discuss in the context of our style guide.